src/main/java/org/cascadebot/cascadebot/commands/informational/UserInfoCommand.java

```diff
@@ -73,10 +79,19 @@ public void onCommand(Member sender, CommandContext context) {
         builder.addField("User ID", userForInfo.getId(), true);
         builder.addField("Mutual Servers", String.valueOf(userForInfo.getMutualGuilds().size()), true);
 
-
+        if (member != null) {
+            PermissionsManager permissionsManager = new PermissionsManager();
+            long ids = Long.parseLong(userForInfo.getId());
+            SecurityLevel userSecurityLevel = permissionsManager.getUserSecurityLevel(ids);
+            if (userSecurityLevel != null) {
+                builder.addField("Official Roles", userSecurityLevel.name(), true);
```

Comment:
Two requested changes here.

This only returns their highest role with the official roles so it's more like position on the project project then the list of official roles they have. So please find a better name for this.

Please properly capitalize the security level. I forget the method to do it off the top of my head, but you can probably find it in the module commands.

Reply:
:+1:

Reply:
`module.toString().toLowerCase()`